test/adapter/json_api/relationship_test.rb

```diff
@@ -1,190 +1,162 @@
 require 'test_helper'
```

Comment:
This file was replaced by `test/adapter/json_api/api_objects/relationship_test.rb` . Nothing to see here.